fnr.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "An open source tool to find and replace text in multiple files.",
+    "homepage": "https://github.com/zzzprojects/findandreplace",
+    "version": "1.8.1",
+    "license": "MIT",
+    "url": "http://findandreplace.io/downloads/fnr.zip",
+    "bin": "fnr.exe",
+    "hash": "205f31941ee62b9b66a5dcbe3c3af5449d54e80095e7304675660951f7fbd4bf",
+    "checkver": {
+        "url": "https://github.com/zzzprojects/findandreplace/releases/",
```

Comment:
Change it to `"github": "https://github.com/zzzprojects/findandreplace"`. No need to append releases.

Reply:
apparently checkver doesn't like that.
```
fnr: couldn't match '/releases/tag/V([\d.]+)' in https://github.com/zzzprojects/findandreplace/
```

Reply:
![proof](https://i.imgur.com/eTtqzSs.png)